src/tracks/InstrumentTrack.cpp

```diff
@@ -1130,6 +1156,37 @@ void InstrumentTrackView::midiConfigChanged()
 
 
 
+QMenu * InstrumentTrackView::createFxMenu(QString title, QString newFxLabel)
```

Comment:
@tresf I put it here because this is a menu that is used to change fx for this Instrument, regardless of if this is used on the mixer view or the lcd spinner box.